src/App.tsx

```diff
@@ -15,35 +16,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Name = 'name',
+  Length = 'length',
+  Empty = '',
+}
```

Comment:
Tips for the future, it is useful to export types to a separate file, for example: `SortType.ts`